extensions/trace_propagators/src/main/java/io/opentelemetry/extensions/trace/propagation/B3PropagatorExtractorMultipleHeaders.java

```diff
@@ -49,8 +49,11 @@
   private static <C> SpanContext getSpanContextFromMultipleHeaders(
       C carrier, HttpTextFormat.Getter<C> getter) {
     String traceId = getter.get(carrier, TRACE_ID_HEADER);
+    if (StringUtils.isNullOrEmpty(traceId)) {
+      return SpanContext.getInvalid();
+    }
     if (!Common.isTraceIdValid(traceId)) {
-      logger.info(
+      logger.warning(
```

Comment:
the other propagators log with `info`
```suggestion
      logger.info(
```

Reply:
Maybe we should change them? If there is a non-empty header, but its format is wrong, shouldn't this be WARN?

Reply:
If upstream sends a bad header, I'd be unhappy to have warning logs spammed without much I can do about it. In general, I'd say tracing should avoid spamming logs as much as possible (even info is a bit questionable, warning much more).

Reply:
But without logs, how do you know about that bad header?

Reply:
I'd generally recommend metrics or using a lower than default level which can be enabled for tests.

Log spam does have reasonable potential to bring down an app and no one likes it when observability kills the app.

Reply:
Yes, definitely. If an unknown upstream actor can creating an arbitrary amount of log spam just by setting some bad headers, that's a DOS attack waiting to happen.

Reply:
Changed all log levels to `fine`